ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,62 @@
+----------------------------- Delete below -----------------------------

```

Comment:
I'm happy to approve as-is, as I think it presents the state of affairs for people who actually read it, but maybe right at the top you could add something like the below to direct people to the proper spot for 50% of the silly issues that are opened?  I should have thought of it on my first comment....

````
Have a general question?
Don't know how to do something?
Trouble with a third-party library?
Please go to https://www.esp8266.com for the support forum to get answers!

Only continue with this issue report if you have a failure or bug with the current core libraries (i.e. crashes, hangs, etc.)
````